File: javascript/1132.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split("\n");
var a = parseInt(lines.shift());
var b = parseInt(lines.shift());
var maior = Math.max(a, b);
var menor = Math.min(a, b);
var total = 0;
for (let i = menor; i <= maior; i++) {
    if(i % 13 != 0){
        total+=i;
    }
}
console.log(total);

File: C/1132.c
#include <stdio.h>

int main() {
    
    int a, b, maior, menor, total = 0, i;
    scanf("%d %d", &a, &b);
    
    if(a > b) {
        maior = a;
        menor = b;
    } else {
        maior = b;
        menor = a;
    }

    for (i = menor; i <= maior; i++) {
        if(i % 13 != 0){
            total += i;
        }
    }
    
    printf("%d\n", total);
    
    return 0;
}